vizro-ai/docs/pages/user_guides/model_config.md

```diff
@@ -45,4 +45,32 @@ cost. In general, the `gpt-3.5-turbo` collection provides the most cost-effectiv
 which would be a good starting point for most users. `gpt-4` models are more powerful than `gpt-3` models, e.g. they allow for more tokens per request. You can refer to these models' [capabilities](https://platform.openai.com/docs/models/overview)
 and [pricing](https://openai.com/pricing) for more information.
 
+### Azure OpenAI models
+
+The setup below allows you to integrate Azure's OpenAI models with VizroAI, by simply updating the `llm_to_use` attribute in `VizroAI`.
+To set up Azure OpenAI with Langchain, you'll configure the AzureOpenAI instance by specifying your deployment name and model name.
+Environment variables for API configuration, such as `OPENAI_API_TYPE`, `OPENAI_API_VERSION`, `OPENAI_API_BASE` and `OPENAI_API_KEY`.
```

Comment:
Not sure I understand this? Is this part of the sentence above? :D or is there a part missing?

"Environment variables for API configuration, such as `OPENAI_API_TYPE`, `OPENAI_API_VERSION`, `OPENAI_API_BASE` and `OPENAI_API_KEY`."

Reply:
ah yes! thank you for catching this. fixed now.